content/integrations/cloudhealth.md

```diff
@@ -15,6 +15,10 @@ With Datadog and CloudHealth, you can:
 * Tackle Reserved Instances with confidence using CloudHealth's analysis and automation
 * Visualize your assets across clouds aligned to line of business
 
+<div class="alert alert-warning">
+No data is collected from Cloudhealth for your Datadog application, Cloudhealth actually polls your infrastructure metrics to be used as part of their Rightsizing analysis.
```

Comment:
I just think the whole Overview needs to be rewritten, and needs to be very short.

"If you use both Cloudhealth and Datadog, you can configure your Cloudhealth account to collect per-instance resource usage metrics from Datadog. This helps CloudHealth gives you more accurate recommendations for adjusting your cloud resources.

This integration does NOT pull anything from Cloudhealth into Datadog. It just helps Cloudhealth poll your Datadog account for metrics."

Let's remove the four bullet points. They're wordy and sound like a sales pitch for Cloudhealth, which is a weird thing for us to have on our Docs site.

And, again, no warning div, please. :)